docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 services:
   biblioteca:
```

Comment:
```
    depends_on:
      db:
        condition: service_healthy
```
I think it would avoid issue to wait for the DB service to be ready.

Reply:
I added it with `service_started` as `service_healthy` requires the container to provide healthchecks which they don't.
Although if you refer to my mention that the build fails because the DB is unavailable this is unrelated as the build doesn't start a DB. I discovered that when you run `cache:cache` if the `DATABASE_URL doesn't contain a `serverVersion` it tries to retrieve it from the DB. adding one solves the issue.

Reply:
Looks good to me thanks for the clarifications.